velox/common/process/ProcessBase.cpp

```diff
@@ -118,6 +119,42 @@ bool hasBmi2() {
   return bmi2CpuFlag && FLAGS_bmi2;
 }
 
+std::mutex TraceContext::mutex_;
```

Comment:
A better way is to use a local static variable in a function in anonymous namespace.